msg/EntityInfo.msg

```diff
@@ -21,3 +21,6 @@ string[] flags
 geometry_msgs/Point[] convex_hull
 float32 z_min
 float32 z_max
+
+# Areas
```

Comment:
Ik vind area's geen geschikte naam. In ED is het ooit zo genoemd (ik denk omdat het om de vloer rondom een entity ging). Het zijn echter volumes en geen oppervlaktes. Vandaar dat dit in robot_skills ook 'volumes' zijn genoemd. Ik zou dat nu ook hier doorvoeren

Reply:
Applied